docs/rest/ReindexFlowResourceBase.http

```diff
@@ -0,0 +1,167 @@
+# This test flow confirms that reindexing operations can handle search parameters that
+# have base type = "Resource", meaning that they can operate on all types of resources.
+# It first creates two test resources: a patient and an observation.
+# Next, it creates a search parameter resource and triggers a reindexing operation.
+# Finally, it uses the new search parameter to confirm the search parameter
+# is enabled and the resources were successfully reindexed.
+# Note: this assumes that appsettings.json has Security.Enabled = false and Reindex.Enabled = true
+
+@baseUrl = https://localhost:44348
+@contentType = application/json
+@patientId = 1407ee02-d505-49fe-9f6f-5fddcf9985f5
+@observationId = da1be095-0a16-481a-a821-e85d676c466e
+@reindexJobId = 438fc1ad-d022-4046-9a57-571488ac1426
+
+###
+# Create a patient resource.
+# Copy the patient's ID in the response body and paste it in the patientId variable.
+POST {{baseUrl}}/Patient HTTP/1.1
```

Comment:
These scripts look good. Do you think we need to consider FHIR versioning at all? i.e. like we do for test data and have /Normative/*.http, or mention in the filename or other?

Reply:
As discussed, I added the compatible versions in a comment in 53cf946.